performance-tuning-methods.md

```diff
@@ -216,34 +216,92 @@ In this workload, only `ANALYZE` statements are running in the cluster:
 - The total number of KV requests per second is 35.5 and the number of Cop requests per second is 9.3.
 - Most of the KV processing time is spent on `Cop-internal_stats`, which indicates that the most time-consuming KV request is `Cop` from internal `ANALYZE` operations.
 
```

Comment:
PR preview: https://pingcap-docsite-preview-git-pr-qiancais-projects.vercel.app/tidb/dev/performance-tuning-methods#cpu-and-memory-usage